Firebase/Storage/FIRStorageUpdateMetadataTask.m

```diff
@@ -52,30 +59,34 @@ - (void)enqueue {
   _completion = nil;
 
   GTMSessionFetcher *fetcher = [self.fetcherService fetcherWithRequest:request];
-  fetcher.comment = @"UpdateMetadataTask";
-  [fetcher beginFetchWithCompletionHandler:^(NSData *data, NSError *error) {
+  _fetcher = fetcher;
+  
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Warc-retain-cycles"
+  _fetcherCompletion =^(NSData *data, NSError *error) {
     if (error) {
       if (!self.error) {
         self.error = [FIRStorageErrors errorWithServerError:error reference:self.reference];
       }
       if (callback) {
         callback(nil, self.error);
       }
+      _fetcherCompletion = nil;
       return;
     }
 
     NSDictionary *responseDictionary = [NSDictionary frs_dictionaryFromJSONData:data];
     if (responseDictionary) {
       FIRStorageMetadata *metadata =
-          [[FIRStorageMetadata alloc] initWithDictionary:responseDictionary];
+      [[FIRStorageMetadata alloc] initWithDictionary:responseDictionary];
```

Comment:
My XCode screws this up all the time. I fixed most of them but guess I missed these.